cps/main.c

```diff
@@ -575,6 +569,12 @@ static int
 assign_cps_queue_ids(struct cps_config *cps_conf)
 {
 	int ret;
+	unsigned int total_pkt_burst = (cps_conf->net->front.rss ?
+		cps_conf->front_total_pkt_burst : 0) +
+		(cps_conf->net->back.rss ? cps_conf->back_total_pkt_burst : 0);
+
+	cps_conf->num_mbuf = calculate_mempool_config_para("cps",
```

Comment:
Field `cps_conf->num_mbuf` is only needed within `assign_cps_queue_ids()`, so why isn't this field a local variable?

This comment applies to the other blocks as well.